src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -259,6 +259,11 @@ public X509Certificate2Collection ClientCertificates
         {
             get
             {
+                if (_clientCertificateOption != ClientCertificateOption.Manual)
```

Comment:
Also, it would need to be separate because someone could use WinHttpHandler separately from HttpClientHandler.  So, we still changes in HttpClientHandler.*.cs specifically.  The only question here is whether we should also change WinHttpHandler to have "similar" behaviors.